package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "plexus",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "jscs": "~2.5.1",
+    "jshint": "~2.9.1-rc1"
+  },
+  "devDependencies": {
+    "jshint": "~2.9.1-rc1",
+    "jscs": "~2.5.1"
```

Comment:
Interesting. I didn't look at the file itself. I just did `npm install --save-dev jshint jscs`, which I assumed would only put stuff in `devDependencies`.